src/collections/graphql/init.ts

```diff
@@ -179,6 +181,35 @@ function registerCollections(): void {
       resolve: deleteResolver(collection),
     };
 
+    if (collection.config.versions) {
+      collection.graphQL.versionType = this.buildVersionType(collection.graphQL.type);
+      this.Query.fields[`version${formatName(singularLabel)}`] = {
+        type: collection.graphQL.versionType,
+        args: {
+          id: { type: GraphQLString },
+        },
+        resolve: findVersionByID(collection),
+      };
+      this.Query.fields[`versions${pluralLabel}`] = {
+        type: buildPaginatedListType(`versions${formatName(pluralLabel)}`, collection.graphQL.versionType),
+        args: {
+          where: { type: buildVersionWhereInputType(singularLabel, collection.config) },
```

Comment:
Might want to double-check all of these arguments against the equivalent `requestHandler` arguments, as `depth` is also missing here